cumulusci/conftest.py

```diff
@@ -107,3 +107,14 @@ def open_db():
                 yield connection, Base.metadata, session
 
         yield open_db
+
+
+@pytest.fixture()
+def delete_data_from_org(create_task):
+    def delete_data_from_org(object_names):
+        from cumulusci.tasks.bulkdata.delete import DeleteData
+
+        t = create_task(DeleteData, {"objects": object_names})
+        t()
+
+    return delete_data_from_org
```

Comment:
This seems like a really weird way to do this. It's possible to get the project config from the org config (org_config.project_config) so it should be pretty straightforward to write a function which takes an org_config and object_names as parameters but isn't tied to the particular way that orgs get set up in the test infrastructure.

Reply:
@prescod I suppose we both share the goal of being able to write tests that are easy to understand, and just have different ideas about the best way to do that. I guess it bothers me because I had to study this fixture for a number of minutes to make sure I understood how it worked. It's not clear that my suggestion is better so I'll drop it for now.